README.md

```diff
@@ -1,111 +1,41 @@
 # Jekyll
 
 ## Installation
-check [here] (https://jekyllrb.com/docs/installation/)
-
-## Mac
-
-### Ruby
-to install *Ruby* you should install RVM, this will install rvm with the latest stable version of Ruby
-```bash
-\curl -sSL https://get.rvm.io | bash -s stable --ruby
-```
-
-If you have problems maybe you need to install the xcode command Line tools
-
-```bash
-xcode-select --install
- ```
-
-### Python
- just use the installer from here https://www.python.org/downloads/mac-osx/
-use version 2.7.x
-
-### NodeJS
-If you don't have it already install Brew
-```bash
-/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-```
-afterwards use brew to install NodeJS
-```bash
-brew install node
-```
-
-### Jekyll
-jekyll is just a gem, a package in Ruby, to resolve all necessary dependencies you should use bundler
-```bash
-gem install bundler
-```
-afterwards install jekyll via bundler, open the blog root directory and
-```
-bundle install
-```
-
-*You should be good to go!!!*
-
-## Ubuntu
-
-### Ruby
-```bash
-\curl -sSL https://get.rvm.io | bash -s stable --ruby
-```
-
-### Python
-Well - should be there...
-
-### NodeJS
-```bash
-sudo apt-get update
-sudo apt-get install nodejs
-sudo apt-get install npm
-```
-
-### Jekyll
-jekyll is just a gem, a package in Ruby, to resolve all necessary dependencies you should use bundler
-```bash
-gem install bundler
-```
-afterwards install jekyll via bundler, open the blog root directory and
-```
-bundle install
-```
-
-*You should be good to go!!!*
-
-## Windows
-
-Install nodejs (https://nodejs.org/en/download/)
-Install ruby (http://rubyinstaller.org/downloads/)
-Install ruby devkit
-- downloaded and extract devkit from http://rubyinstaller.org/downloads/ to c:\rubyx.y\devkit
-- navigate to c:\rubyx.y\devkit fire 'ruby dk.rb init' and 'ruby dk.rb install'
-
-Navigate to blog-clone and invoke 'bundle install' and 'bundle exec jekyll serve --config _config_local.yml'
-Go to http://localhost:4000
-
-
-## branching
-
-### master
+
```

Comment:
I would also leave the old school integration for the geeks here that don't want to install docker and already have everything else setup.